src/app/Dashboard/components/DashboardContainer.js

```diff
@@ -45,22 +48,38 @@ export class DashboardContainer extends Component {
     pointsEarned: PropTypes.number,
     activitiesLogged: PropTypes.number,
     fetchUserActivites: PropTypes.func,
+    loadCategories: PropTypes.func,
     userActivities: PropTypes.arrayOf(PropTypes.shape({})),
   };
 
   componentDidMount() {
-    const { fetchUserActivites } = this.props;
+    const { fetchUserActivites, loadCategories } = this.props;
     const token = getToken();
     const userInfo = getUserInfo(token);
     this.setState({ user: userInfo });
     fetchUserActivites(userInfo.id);
+    loadCategories();
+  }
+
+  openModalLoginPointsHandler = () => {
```

Comment:
This function does the same thing as the function on line 70, I think you can have one function it will also help you have an easier name `openModalLoginPointsHandler` hehe

Reply:
@Kachulio1 I have refactored this. I only have one function for this.